.github/workflows/auto_test.yml

```diff
@@ -87,3 +90,84 @@ jobs:
           fi
         done
+
+  test-lite-card:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      with:
+        repository: CoolBitX-Technology/coolwallet-sdk
+        path: sdk
+        ref: ${{ github.event.pull_request.head.ref }}
+        submodules: recursive
+        token: ${{ secrets.DEVOPS_READ_PACKAGE_TOKEN }}
+
+    - name: Authenticate gcloud
+      uses: google-github-actions/auth@v2
+      with:
+        credentials_json: ${{ secrets.DEV_CP_GCP_SVC_GITHUB_RUNNER_KEY_FILE }}
+
+    - name: Fetch base branch from upstream repository
+      run: |
+        cd ${{ github.workspace }}/sdk
+        git remote add upstream https://github.com/CoolBitX-Technology/coolwallet-sdk.git
+        git fetch upstream ${{ github.event.pull_request.base.ref }}
+
+    - name: Get list of modified files and check for modified package.json in coin directories
+      id: modified_files
+      working-directory: sdk
+      run: |
+        # cd ${{ github.workspace }}/sdk
+        MODIFIED_FILES=$(git diff --name-only upstream/${{ github.event.pull_request.base.ref }})
+        echo "$MODIFIED_FILES"
+
+        MODIFIED_COINS=""
+        for file in $MODIFIED_FILES; do
+          echo "file = $file"
+          if [[ $file == packages/coin-*/package.json ]]; then
+            COIN_DIR=$(dirname "$file")
+            MODIFIED_COINS="$MODIFIED_COINS $COIN_DIR"
+          fi
+        done
+
+        MODIFIED_COINS=$(echo $MODIFIED_COINS | xargs)  # Trim leading/trailing whitespace
+        echo "modified_coins=$MODIFIED_COINS" >> $GITHUB_OUTPUT
+
+    - name: Set up Cloud SDK
+      uses: google-github-actions/setup-gcloud@v2
+
+    - name: Docker auth
+      run: gcloud auth configure-docker ${{ env.REGISTRY }}
+
+    - name: Pull Docker image
+      run: docker pull ${{ env.REGISTRY }}/${{ secrets.DEV_CP_GCP_PROJECT_ID }}/coolwallet-lite/jcvm
+
+    - name: Run Docker container
+      run: docker run -e CARD=lite -d --name coolwallet-jcvm-jre-http -p 9527:9527 ${{ env.REGISTRY }}/${{ secrets.DEV_CP_GCP_PROJECT_ID }}/coolwallet-lite/jcvm
+
+    - name: Dump node environment
+      working-directory: sdk
+      run: 'echo "node: $(node -v) npm: $(npm -v)"'
+
+    - name: Install node modules
+      working-directory: sdk
+      run: npm install
+
+    - name: Set up test fixture
+      working-directory: sdk
+      run: sh scripts/install-test-fixture.sh
+
+    - name: Run tests for all modified coin project
+      run: |
+        for modified_coin in ${{ steps.modified_files.outputs.modified_coins }}; do
+          if [ -f "${{ github.workspace }}/sdk/$modified_coin/tests/index.spec.ts" ]; then
+            echo "Test file found for = $modified_coin"
+            cd ${{ github.workspace }}/sdk/$modified_coin
+            npm install
+            npm run ci-test-lite
+          else
+            echo "Test file not found for $modified_coin"
+          fi
+        done
```

Comment:
The new line at end of file is missing

Reply:
[55cad69](https://github.com/CoolBitX-Technology/coolwallet-sdk/pull/904/commits/55cad69fcc1ee651099d2329c722c7419780103b)